File: auto-docs/executables/julia/axes_range_manual.jl
using Plotly
Plotly.signin("TestBot", "r1neazxo9w")

trace1 = [
  "x" => [0, 1, 2, 3, 4, 5, 6, 7, 8], 
  "y" => [8, 7, 6, 5, 4, 3, 2, 1, 0], 
  "type" => "scatter"
]
trace2 = [
  "x" => [0, 1, 2, 3, 4, 5, 6, 7, 8], 
  "y" => [0, 1, 2, 3, 4, 5, 6, 7, 8], 
  "type" => "scatter"
]
data = [trace1, trace2]
layout = [
  "xaxis" => ["range" => [2, 5]], 
  "yaxis" => ["range" => [2, 5]]
]
response = Plotly.plot(data, ["layout" => layout, "filename" => "axes-range-manual", "fileopt" => "overwrite"])
plot_url = response["url"]

File: auto-docs/executables/r/axes_range_manual.r
library(plotly)
p <- plotly(username='TestBot', key='r1neazxo9w')

trace1 <- list(
  x = c(0, 1, 2, 3, 4, 5, 6, 7, 8), 
  y = c(8, 7, 6, 5, 4, 3, 2, 1, 0), 
  type = "scatter"
)
trace2 <- list(
  x = c(0, 1, 2, 3, 4, 5, 6, 7, 8), 
  y = c(0, 1, 2, 3, 4, 5, 6, 7, 8), 
  type = "scatter"
)
data <- list(trace1, trace2)
layout <- list(
  xaxis = list(range = c(2, 5)), 
  yaxis = list(range = c(2, 5))
)
response <- p$plotly(data, kwargs=list(layout=layout, fileopt="overwrite", filename="axes-range-manual"))
url <- response$url